hend.py

```diff
@@ -30,17 +23,13 @@ def feature_extract(df):
     IQR = Q3 - Q1
 
     #I replaced outliers with the mean.
-    for entry in df['LotFrontage']:
-        if entry > (Q3 + 1.5 * IQR):
-            df['LotFrontage'].replace(entry, df['LotFrontage'].mean(), inplace = True)
+    df['LotFrontage'].clip(0,((Q3 + 1.5 * IQR))

```

Comment:
You have a hanging paren '(' after clip that isn't resolved. It looks like your goal is to clip the data to either zero or the 3rd quartile plus 1.5 IQR's, right? That looks good. But you might want to drop one of the '(' before Q3. I think that will solve your problem.